Dataset.py

```diff
@@ -277,11 +289,13 @@ def get_seq_length_nd(self, sorted_seq_idx):
 
   def get_seq_length(self, seq_idx):
     """
+    TODO: This function seems to be replaced in all existing subclasses, so the body might be removed here.
```

Comment:
Are you sure? I see only 6 overwrites. But there are far more subclasses.

Reply:
It is overwritten in CachedDataset, CachedDataset2, GeneratingDataset, MetaDataset and SprintBaseDataset, which covers all Datasets that I found so far. And only in CachedDataset there is a reference to `self.get_seq_length_nd`, which is why I thought this is not needed in the Dataset anymore

Reply:
There are about 20-30 datasets.

Reply:
Yes, and all are based on the 6 Mentioned. So the function from Dataset itself is never called, also not via super()

Reply:
Well, if you are sure (I did not check, but I assume you know how to check this), you can remove this dummy code, or rather move to `CachedDataset`, and put a `raise NotImplementedError` here.